domains/certificates/Query.API/API/MeasurementsSyncer/SlidingWindowService.cs

```diff
@@ -27,19 +21,29 @@ public MeteringPointTimeSeriesSlidingWindow CreateSlidingWindow(Gsrn gsrn, UnixT
         return MeteringPointTimeSeriesSlidingWindow.Create(gsrn, synchronizationPoint, missingMeasurements);
     }
 
-    public List<Measurement> FilterMeasurements(MeteringPointTimeSeriesSlidingWindow window, List<Measurement> measurements)
+    public List<Measurement> FilterMeasurements(MeteringPointTimeSeriesSlidingWindow window, List<Measurement> measurements, int minimumAgeBeforeIssuingInHours)
```

Comment:
I believe this method handles filters measurements we do not expect to be able to handle, even in a later run. But the measurements filtered in the new method, those we explicitely expect to handle in a later run. 

Does this work? I don't think i see any new tests verifying this?